packages/server/lib/browsers/utils.coffee

```diff
@@ -7,6 +7,12 @@ profileCleaner = require("../util/profile_cleaner")
 
 PATH_TO_BROWSERS = appData.path("browsers")
 
+getBrowserPath = (browser) ->
```

Comment:
after investigating, there's no reason to namespace this further based on the way we handle profile creation and partitions